esp/esdllib/esdl_xpath_extensions_libxml.cpp

```diff
@@ -198,9 +367,19 @@ void registerEsdlXPathExtensions(IXpathContext *xpathContext, IEspContext *conte
         xpathContext->registerFunction(nullptr, "validateFeaturesAccess", (void  *)validateFeaturesAccessFunction);
         xpathContext->registerFunction(nullptr, "secureAccessFlags", (void  *)secureAccessFlagsFunction);
         xpathContext->registerFunction(nullptr, "getFeatureSecAccessFlags", (void  *)getFeatureSecAccessFlagsFunction);
+        xpathContext->registerFunction(nullptr, "getStoredStringValue", (void  *)getStoredStringValueFunction);
+        xpathContext->registerFunction(nullptr, "storedValueExists", (void  *)storedValueExistsFunction);
+        xpathContext->registerFunction(nullptr, "getLogProfile", (void  *)getLogProfileFunction);
+        xpathContext->registerFunction(nullptr, "getLogOption", (void  *)getLogOptionFunction);
+        xpathContext->registerFunction(nullptr, "logOptionExists", (void  *)logOptionExistsFunction);
```

Comment:
Because I know the implementation details I could use this to test for a selected log profile, but should I have to? It seems like logProfileExists should be defined.

Reply:
You could just check if getLogProfile returns content.  I'll think about it.

Reply:
Running out of time for the initial 7.10 RC.  We can add a new xpath function for this later.  Just get if getLogProfile has content for now.